src/olympia/editors/tests/test_views.py

```diff
@@ -2999,6 +3001,9 @@ def test_auto_approval_summary(self):
             'Has too few consecutive human-approved updates.')
         assert (
             doc('.auto_approval li').eq(1).text() ==
+            'Has too many daily users.')
+        assert (
+            doc('.auto_approval li').eq(2).text() ==
             'Uses a custom CSP.')
```

Comment:
The issue was that we could not display "Has too many daily users" reason, because the extra comma made the comparison wrong. That's why I added `average_daily_users=10000` when creating the `AutoApprovalSummary` in this test. Without the fix, the revised version of the test would not pass